Example/Example/EditorDemoController.swift

```diff
@@ -158,22 +170,42 @@ class EditorDemoController: UIViewController {
 
     }
 
+    // MARK: - Title and Title placeholder position methods
+    func updateTitleHeight() {
+        let sizeThatShouldFitTheContent = titleTextField.sizeThatFits(CGSize(width:self.view.frame.size.width - ( 2 * Constants.margin), height: CGFloat.greatestFiniteMagnitude))
```

Comment:
Maybe move the CGSize instantiation one line upwards, to aid readability? 
+ No need to use `self` nor `size`. `view.frame.width` should do the trick!